packages/web/src/components/earn/incentivized-pool-card/IncentivizedPoolCard.tsx

```diff
@@ -18,16 +17,17 @@ export interface IncentivizedPoolCardProps {
   pool: PoolCardInfo;
   routeItem: (id: string) => void;
   themeKey: "dark" | "light";
+  isStakedPool: (poolPath: string | null) => boolean
```

Comment:
It's confusing that `isStakedPool` is a function. 
It's certainly not misnamed, but we already use boolean types for `isLoading`, `isPoolFetched`, etc. 
How about changing it to `checkStakedPool`?

Reply:
i will fix this